packages/server/db/index.js

```diff
@@ -0,0 +1,33 @@
+const levelup = require("levelup");
+const leveldown = require("leveldown");
+const { ethers } = require("ethers");
+const path = require("path");
+
+const dbPath = path.join(__dirname, "tokens");
+
+const tokens = levelup(leveldown(dbPath));
+
+const getRandomTokenId = () => ethers.BigNumber.from(ethers.utils.randomBytes(32)).toString();
+
+exports.getOrStoreRandomId = async (url) => {
```

Comment:
I'm just using a random uint256 generator and a simple key,value store to keep track of token ids; open to suggestions/ideas/feedback here!